indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -1251,12 +1252,19 @@ private TaskStatus runSequential(TaskToolbox toolbox) throws Exception
    */
   private TaskReport.ReportMap getTaskCompletionReports(TaskStatus taskStatus)
   {
-    return buildIngestionStatsAndContextReport(
+    final var taskCompletionReport =  buildIngestionStatsAndContextReport(
```

Comment:
Druid code doesn't really use the `var` keyword yet. So, it would be nice to stick to static typing for the time being.